lib/adb/adbcontroller.py

```diff
@@ -13,4 +13,24 @@ def run_command(id, *command):
         return output
 
 
+    def make_phone_call_adb(self,device_id,number):
+        """
+        It receives a telephone number and makes the call with adb
+        Pre-condition: the number has to be valid
+        param str numner: telephone number
+        format number: "1234567890"
+        returns: 
+        """
+        """
+        validate the format and the lenght of the number
+        """
+        Phone_number = ''.join(i for i in number if i.isdigit())
+        if len(Phone_number) != 10:
+            print("Formato incorrecto, tiene que tener un tamaño de 10 digitos: 4491234567")
+            return False
```

Comment:
Sería mejor levantar una excepción?

Reply:
```suggestion
            Phone_number =''.join(i for i in number if i.isdigit() or i=="+")
        if len(Phone_number) == 13:
            return True
        else:
            raise ValueError("Incorrect Format number")
```

esta bien esta excepcion?